src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/repository/ConfigurationSourceRepository.java

```diff
@@ -28,11 +28,14 @@
 
 import org.niis.xroad.cs.admin.core.entity.ConfigurationSourceEntity;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface ConfigurationSourceRepository extends GenericRepository<ConfigurationSourceEntity, Integer> {
```

Comment:
There is something wrong here. We have two methods:
1. Optional<ConfigurationSourceEntity> findBySourceType(String source)
2. List<ConfigurationSourceEntity> findAllBySourceType(String source);

If there can be multiple ConfigurationSourceEntity, the first one will fail when more than one available.
If there always will be only one ConfigurationSourceEntity, the findAllBySourceType(..) is not needed.